test/op-worker-test.sh

```diff
@@ -1,7 +1,12 @@
 #!/bin/bash -e
 
+if [[ -v "$DD_OPW_INSTALL_CLASSIC_AGENT" ]]; then
+$DD_OPW_INSTALL_CLASSIC_AGENT
+fi
+
 $SCRIPT
 
+
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ Stray linebreak added?
```suggestion
```